tests/src/tests/happy.rs

```diff
@@ -29,6 +32,9 @@ async fn happy_test(nodes: Vec<NomosNode>) {
         } => {}
     };
 
+    let elapsed = start_time.elapsed();
+    println!("ELAPSED: {elapsed:?}");
```

Comment:
> Weirdly, this is much faster than the test without mixnet (5s). I'm taking a look at why.

I found that it's because the mixclient always pick a mix destination from the exit layer, and there is one mixnode in the exit layer in our test setting. It means that all messages are eventually broadcasted from only one node.
If I change mixclient to pick any mixnode as a mix destination regardless of the layer, it produces the latency 6s, which looks reasonable compared to the test without mixnet.

I still don't understand why the performance is good when all messages are broadcasted from only one nomos-node. I'll take a look.

Reply:
added a document in notion: https://www.notion.so/Mixnet-Measurements-551ade11ae4d44ca9f3d947ea6950c67